src/edge_utils.c

```diff
@@ -347,12 +347,6 @@ void supernode_disconnect (n2n_edge_t *eee) {
     if(eee->sock >= 0) {
         closesocket(eee->sock);
         eee->sock = -1;
-
```

Comment:
Please note that when exiting the edge, don't forget to clean up the port mapping.